4.0.6 release

finnlewis opened the pull request.

stephen-cox committed:
Added tests for setting domain path aliases #530

finnlewis committed:
Merge pull request #533 from localgovdrupal/fix/4.x/530-tests-for-paths

Added tests for setting domain path aliases

AWearring committed:
Patch: Undefined array key 'pathauto' no longer applies

Removing `domain_path` patch `"Warning: Undefined array key 'pathauto' #3315752": "https://www.drupal.org/files/issues/2022-10-17/undefined-array-key-pathauto-3265497-2.patch"` and bumping `domain_path` version to 1.5

finnlewis committed:
Merge pull request #541 from localgovdrupal/fix-4_x-patch-ndefined-array-key-pathauto-no-longer-applies

Patch: Undefined array key 'pathauto' no longer applies

finnlewis committed:
Fix the geo permissions for a clean install.

finnlewis committed:
Merge pull request #544 from localgovdrupal/feature/543-fix-permissions

Fix the geo permissions for a clean install.

ekes committed:
Disable per-domain access control while indexing. (#545)

This populates the index with all items and their referenced items.
The items are filtered on output.

See https://github.com/localgovdrupal/localgov_microsites/issues/524

ekes commented:
```
1) Drupal\Tests\localgov_microsites_group\Functional\GroupInvitationAccessTest::testInvitationPermissions
Behat\Mink\Exception\ExpectationException: Current response status code is 403, but 200 expected.
```

ekes committed:
Add new permissions to admin user for Group Invite (4.0.0) Fixes #549 (#550)

Permissions applied the same as the would be to existing sites by the
module upgrade.

finnlewis commented:
```

1) Drupal\Tests\localgov_microsites_group\Functional\MicrositeCreationTest::testMicrositeCreationForm
Behat\Mink\Exception\ResponseTextException: The text "@*Wj>&<q" was not found anywhere in the text of the current page.
```


I will endeavour to replicate locally and explore the test.

tonypaulbarker commented:
@finnlewis hooray! We have all tests passing.
@ctorgalson noticed in debugging that there is a problem when the redirect patch doesn't apply https://github.com/localgovdrupal/localgov_microsites_group/actions/runs/17236308442/job/48905385173?pr=542#step:10:868  but now we have a workaround for the patch we see a clean bill of health on this PR.

finnlewis commented:
Brilliant! Merging and tagging.... thanks @tonypaulbarker @ctorgalson  :)
